.github/CODEOWNERS

```diff
@@ -48,3 +48,8 @@
 # app/src/main/java/com/x8bit/bitwarden/ui/vault  @bitwarden/team-vault-dev
 # app/src/test/java/com/x8bit/bitwarden/data/vault @bitwarden/team-vault-dev
 # app/src/test/java/com/x8bit/bitwarden/ui/vault  @bitwarden/team-vault-dev
+
+# Default shared ownership of Dockerfile
+**/Dockerfile @bitwarden/team-appsec @bitwarden/dept-bre
+**/*.dockerignore @bitwarden/team-appsec @bitwarden/dept-bre
+**/entrypoint.sh @bitwarden/team-appsec @bitwarden/dept-bre
```

Comment:
We don't use Dockerfiles. Is this being added just to future-proof the repo?

Reply:
@fntyler friendly nudge. This seems unnecessary considering we don't use Dockerfiles. Is this purely a preventative measure?

Reply:
Understood. Thanks for confirming.